packages/editor/src/lib/utils/rotation.ts

```diff
@@ -14,12 +14,16 @@ export function getRotationSnapshot({ editor }: { editor: Editor }) {
 
 	// todo: this assumes we're rotating the selected shapes
 	// if we try to rotate shapes that aren't selected, this
-	// will produce the wrong results if there are other shapes
-	// selected or else break if there are none.
+	// will produce the wrong results
+
+	// Return null if there are no selected shapes
+	if (!selectionPageCenter) {
+		return null
+	}
 
 	return {
-		selectionPageCenter: selectionPageCenter!,
-		initialCursorAngle: selectionPageCenter!.angle(originPagePoint),
+		selectionPageCenter: selectionPageCenter,
```

Comment:
We should compute this based on the shapes being passed in rather than by the selection.

Reply:
Yes, see my note - this PR just prevents the crash, but we should also resolve the todo within this file. Would you rather we just do that now instead?